sass/main.scss

```diff
@@ -940,6 +938,57 @@ footer {
         grid-template-columns: repeat(4, $leadership-size-mobile);
         justify-content: space-between;
     }
+
+}
+
+@media screen and (min-width: 1700px) {
+    nav {
+        font-size: 1.5em;
+
+        #login {
+	    padding: 15px 30px;
+	}
+
+	#logo {
```

Comment:
Hmm, this won't change anything except make the logo not aligned with the rest of the content:
![Screen Shot 2020-12-20 at 1 37 58 AM](https://user-images.githubusercontent.com/2782749/102706906-fedca280-4263-11eb-8085-c58f4ff5a069.png)

Might be better to leave logo (and navbar, as you've already done) at the same size, looks fine imo

Reply:
[removed in this commit](https://github.com/codeRIT/brickhack.io/pull/1053/commits/cb54bb2673be9e092417dba2a09e1423c7d038aa)